### PR TITLE
feat(scheduler-service): SFO-97 change freebusy api logic to accept more any…

### PR DESCRIPTION
**Pull request by IshuSF**
## Description

Now free busy status can be get by applying any of the event attendee view filters.

Fixes # (SFO-97)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] npm test and postman

## Checklist:

- [x] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: services/scheduler-service/src/models/EventAttendeeViewItemDTO.dto.ts [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  startDateTime?: Date;
+
+  @property({
+    type: 'string',
+    jsonSchema: {
+      enum: [
+        StatusType.Confirmed,
+        StatusType.Tentative,
+        StatusType.Cancelled,
+        null,
```
Comment (vineet-suri) [inline services/scheduler-service/src/models/EventAttendeeViewItemDTO.dto.ts:104]: why giving null in enums

Comment (rohit-sourcefuse) [reply services/scheduler-service/src/models/EventAttendeeViewItemDTO.dto.ts:104]: @rashisf maybe this null is going to replace with completed

Comment (rohit-sourcefuse) [reply services/scheduler-service/src/models/EventAttendeeViewItemDTO.dto.ts:104]: @rashisf maybe this null is going to replace with completed

Comment (rashisf) [reply services/scheduler-service/src/models/EventAttendeeViewItemDTO.dto.ts:104]: Yes it would be replaced but if you want I can remove it

**Pull request merged**